docs/docsite/rst/user_guide/playbooks_filters.rst

```diff
@@ -142,10 +142,18 @@ To get the minimum value from list of numbers::
 
     {{ list1 | min }}
 
+To get the minimum value in a list of objects::
+
+    {{ [{'val': 1}, {'val': 2}] | min(attribute='val') }}
+
 To get the maximum value from a list of numbers::
 
     {{ [3, 4, 2] | max }}
 
+To get the maximum value in a list of objects::
+
+    {{ [{'val': 1}, {'val': 2}] | max(attribute='val') }}
+
```

Comment:
Add a note saying `attribute` is added in 2.8 release.

Reply:
Added!